lms/services/assignment.py

```diff
@@ -0,0 +1,67 @@
+from functools import lru_cache
+
+from lms.models import ModuleItemConfiguration
```

Comment:
Do you have a plan to resolve the terminology conflict of `Assignment`s vs `ModuleItemConfiguration`s?

Reply:
Maybe rename the model class at some point. I think renaming the actual DB table might be difficult